ignis/services/fetch/service.py

```diff
@@ -223,6 +224,20 @@ def cpu(self) -> str:
                     break
         return cpu_name
 
+    @GObject.Property
+    def cpu_temp(self) -> float:
+        """
+        - read-only
+
+        Current CPU temperature.
+        """
+        for thermal_zone in glob.glob("/sys/class/thermal/thermal_zone*/temp"):
+            with open(thermal_zone) as f:
```

Comment:
Not all thermal zones refer to CPU
You have to add a check  to see if ``thermal_zone*/type`` is CPU